include/iotc_types.h

```diff
@@ -204,33 +188,31 @@ typedef union iotc_crypto_key_union_u {
 } iotc_crypto_key_union_t;
 
 /**
- * @enum iotc_crypto_key_signature_algorithm_e
- * @brief specifies a key signature algorithm. This may be used to
- * signify a key type, or to specify how a JWT might be signed.
- * This should match the algorithm type that was used to provision the device's
- * public key in Google IoT Core.
+ * @enum  iotc_crypto_key_signature_algorithm_e
+ * @brief Define a key signature algorithm to sign JWTs.
  *
- * The value provided here will also determine the "alg" field of the JWT, which
- * is automatically assembled by the IoTC Client during the connection process.
+ * This value also determines the JWT "alg" field, which the client application
```

Comment:
no longer true. The client application must call the create_iot_core_jwt function in iotc_jwt.h before calling iotc_connect